Core/include/Acts/Utilities/AnnealingUtility.hpp

```diff
@@ -32,14 +32,23 @@ class AnnealingUtility {
     // Config constructor with default temperature list: {64.,16.,4.,2.,1.5,1.}
     Config(const std::vector<double>& temperatures = {64., 16., 4., 2., 1.5,
                                                       1.})
-        : setOfTemperatures(temperatures) {}
+        : setOfTemperatures(temperatures) {
+      // Set Gaussian cut-off terms for each temperature
+      for (double temp : temperatures) {
+        gaussCutTempVec.push_back(std::exp(-cutOff / (2. * temp)));
+      }
+    }
```

Comment:
The vector of pre-computed cutoff values is not a separate configuration setting. It should be a private member in the annealing utility and not set in the configuration. As a further simplification, you can also get rid of the configuration constructor altogether using default initializations, i.e.
```cpp
struct Config {
  double cutOff = 9.;
  std::vector<double> temperatures = { ... };
};
```

Reply:
But it kind of is a computed property of the configuration. I think either is fine.

Reply:
Done.

Reply:
I agree with Paul, that's why I had it in there initially. But I think it's also fine as a member variable. So I'll leave as is now.